src/edu/csus/ecs/pc2/clics/API202306/CLICSRun.java

```diff
@@ -0,0 +1,131 @@
+// Copyright (C) 1989-2025 PC2 Development Team: John Clevenger, Douglas Lane, Samir Ashoo, and Troy Boudreau.
+package edu.csus.ecs.pc2.clics.API202306;
+
+import java.util.Calendar;
+import java.util.Date;
+import java.util.Set;
+import java.util.logging.Level;
+
+import com.fasterxml.jackson.annotation.JsonFilter;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import edu.csus.ecs.pc2.core.IInternalController;
+import edu.csus.ecs.pc2.core.Utilities;
+import edu.csus.ecs.pc2.core.model.ContestTime;
+import edu.csus.ecs.pc2.core.model.IInternalContest;
+import edu.csus.ecs.pc2.core.model.JudgementRecord;
+import edu.csus.ecs.pc2.core.model.Run;
+import edu.csus.ecs.pc2.core.util.IJSONTool;
+
+/**
+ * CLICS Judgment Run
+ * Contains information about a Run
+ *
+ * @author John Buck
+ *
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonFilter("rtFilter")
+public class CLICSRun {
+
+    @JsonProperty
+    private String id;
+
+    @JsonProperty
+    private String submission_id;
+
+    @JsonProperty
+    private String judgement_type_id;
+
+//    Only for "score" type contests, N/A for ICPC pass/fail
+//    @JsonProperty
+//    private String score;
+
+    @JsonProperty
+    private String start_time;
+
+    @JsonProperty
+    private String start_contest_time;
+
+    @JsonProperty
+    private String end_time;
+
+    @JsonProperty
+    private String end_contest_time;
+
+    @JsonProperty
+    private double max_run_time;
+
+    /**
+     * Fill in properties for a Problem description.
+     *
+     * @param model The contest
+     * @param problem The problem
```

Comment:
Now I'm even more confused -- these comments are talking about things related to _problems_.  I'm going to suspend further review of this class until I get some clarification about what's going on...

Reply:
See my explanation above indicating this class corresponds to the CLICS Judgements endpoint.